<?xml version="1.0"?>
<launch>
    <arg name="wheel_diameter" default="0.08" />
    <arg name="base_width" default="0.135" />
    <arg name="encoder_ticks_per_rev" default="3840" />
    <arg name="linear_coef" default="100" />
    <arg name="angular_coef" default="50" />

    <node name="Pynq_Car" pkg="car_driver" type="car_driver.py">
        <param name="wheel_diameter" value="$(arg wheel_diameter)" />
        <param name="base_width" value="$(arg base_width)" />
        <param name="encoder_ticks_per_rev" value="$(arg encoder_ticks_per_rev)" />
        <param name="linear_coef" value="$(arg linear_coef)" />
        <param name="angular_coef" value="$(arg angular_coef)" />
    </node>
</launch>
